src/Compiler/RefC/RefC.idr

```diff
@@ -1073,34 +1051,29 @@ compileExpr ANF c _ outputDir tm outfile =
      coreLift_ $ writeFile outn code
      coreLift_ $ putStrLn $ "Generated C file " ++ outn
 
-     cc <- coreLift findCC
-     dirs <- getDirs
-
-     let runccobj = cc ++ " -c " ++ outn ++ " -o " ++ outobj ++ " " ++
-                       "-I" ++ fullprefix_dir dirs "refc " ++
-                       "-I" ++ fullprefix_dir dirs "include"
+     pure (Just outn)
 
-     let runcc = cc ++ " " ++ outobj ++ " -o " ++ outexec ++ " " ++
-                       fullprefix_dir dirs "lib" </> "libidris2_support.a" ++ " " ++
-                       "-lidris2_refc " ++
-                       "-L" ++ fullprefix_dir dirs "refc " ++
-                       clibdirs (lib_dirs dirs)
+export
+compileExpr : UsePhase
+           -> Ref Ctxt Defs
+           -> (tmpDir : String)
+           -> (outputDir : String)
+           -> ClosedTerm
+           -> (outfile : String)
+           -> Core (Maybe String)
+compileExpr ANF c _ outputDir tm outfile =
+  do let outn = outputDir </> outfile ++ ".c"
+     let outobj = outputDir </> outfile ++ ".o"
+     let outexec = outputDir </> outfile
 
-     coreLift $ putStrLn runccobj
-     0 <- coreLift $ system runccobj
-       | _ => pure Nothing
-     coreLift $ putStrLn runcc
-     0 <- coreLift $ system runcc
-       | _ => pure Nothing
-     pure (Just outexec)
+     _ <- coreLift $ mkdirAll outputDir
```

Comment:
Same question here about changing a call to `coreLift_` to `_ <- coreLift`.

Reply:
Same response.

Reply:
Same response.